### ISSUE TITLE
Photos are not uploaded alphabetically sorted to flickr

### ISSUE THREAD

**Issue by ruzickap** [TheFox/flickr-cli#44]
Hello.

When I add a directory as argument to `flickr-cli` it mix all the photos together and doesn't upload them sorted by name.

For example I have these photos on the disk:

```
# ls -l 2017/2017.09.16-Osowka_Ksiaz/
total 781304
-rw-r--r-- 1 pruzicka pruzicka 13222382 Sep 16  2017 20170916-100951.jpg
-rw-r--r-- 1 pruzicka pruzicka 13803622 Sep 16  2017 20170916-101138.jpg
-rw-r--r-- 1 pruzicka pruzicka 14014644 Sep 16  2017 20170916-101514.jpg
-rw-r--r-- 1 pruzicka pruzicka 13527584 Sep 16  2017 20170916-102144.jpg
```

I would expect, that `flickr-cli` will upload them to flickr sorted (like I can see them on my local disk).

Thank you

**Comment by TheFox** [comment]
This is because you get the files how they are stored on the disk. I'm pretty sure `ls` also does a sort before output.

**Comment by ruzickap** [comment]
Yes... I guess you are right...
Would it be possible add some sort function to the "list/array" of the photos before they are uploaded to Flickr ?

**Comment by TheFox** [comment]
Sure.

**Comment by ruzickap** [comment]
Sorting was fixed by #45 .
Closing...
